share/spice/twitter/twitter.js

```diff
@@ -39,6 +39,20 @@
                 if (item.location) {
                     subtitle.push(item.location);
                 }
+                var follow_cnt = "Followers: ";
+                if(item.followers_count >= 1000000000) {
```

Comment:
This doesn't look necessary. Katy Perry has the most and shes at 97M -- we're quite far from 1B followers ;)